app/javascript/custom/kindergarten.js

```diff
@@ -0,0 +1,48 @@
+var toggle = true;
```

Comment:
Can we use Stimulus instead event listeners ?

Reply:
I can try. But keep in mind that there's a bug in Tailwind when dynamically changing screensize which affects breakpoint classes eg. changing from low to high res dinamically can cause navbar menu to break down (changing from mobile to 768px causes navbar to remain hidden at certain conditions). This is why custom .js is unavoidable.

Reply:
@DeusCorona please create another issue for that bug so we can discuss there